bin/setup_flat

```diff
@@ -72,7 +86,11 @@ container=forch-faucet-$inst
 container_inst=inst/$container/
 mkdir -p $container_inst/
 cp -r topo/$topo/faucet $container_inst/
+setup_flat_faucet_config $switches
+cp topo/common/faucet/ryu.conf $container_inst/faucet/
```

Comment:
this should just be * at the end, not ryu.conf -- reason is to make it transparent to the script since the script...

Reply:
Copying the directory and using * in the command below where we copy the fiels in $topo/faucet directory